clients/ordering/src/index.tsx

```diff
@@ -7,6 +7,7 @@ import {
 
 import './index.css';
 import App from './App';
+import { BrowserRouter } from 'react-router-dom';
 import * as serviceWorker from './serviceWorker';
 
 const theme = createMuiTheme({
```

Comment:
You should merge the latest master into this branch before merging, and import the theme from `@jeeves/common`